test/inspector/test-inspector-port-zero-cluster.js

```diff
@@ -17,14 +18,14 @@ if (cluster.isMaster) {
     }));
     worker.send('debugPort');
   }
-  process.on('exit', () => {
+  process.on('exit', common.mustCall(() => {
```

Comment:
I think this is true iff event handlers are called in the order they were registered. So the the one checking all the `mustCall`s would run before this one, and IIUC it would fail since this instance didn't run yet.

Anyway `test-inspector-port-zero-cluster.js` is a PITA, re: https://github.com/nodejs/node/pull/13373